### ISSUE TITLE
Sprint status seems wrong

### ISSUE THREAD

**Issue by SettingDust** [theendercore/VisibleToggleSprintMulti#2]
Ctrl will switch to sprint when pressing w(forward). The icon shows only when pressing ctrl but not pressing w or double click w that is spriting

**Comment by theendercore** [comment]
This has been a know thing since the mod was only fabric ([#1](https://github.com/theendercore/VisibleToggleSprint/issues/1)) and I just have forgotten to add this. But since I need to update to 1.20.3 I'll add this to my to do list.

Also sorry I didn't respond sooner. GitHub notifications suck and i don't check my email.

**Comment by theendercore** [comment]
Okay so I looked at this and I now understand why I did it this way. If I where to check if the player was only sprinting or crouching it would Not be and indicator of if the the sprint / crouch is toggled. It would only show the state in which the player is right now. And that was not the original goal of the mod. But Ill just add this as a toggle in the settings (on by default) since I do think it is a nice feature and the mod has evolved way past its original goal.
